libs/npm/core/src/components/form/options/interfaces/OptionItem.ts

```diff
@@ -6,21 +6,21 @@ export class OptionItem<T extends string | number | undefined = string | number
   discriminator: 'IOption';
   label: string | React.ReactElement;
   value: T;
-  isEnabled: boolean;
+  isDisabled: boolean;
 
   constructor(label: string | React.ReactElement, value: T, isEnabled: boolean = true) {
     this.discriminator = 'IOption';
     this.label = label;
     this.value = value;
-    this.isEnabled = isEnabled;
+    this.isDisabled = isEnabled;
   }
 
   static create<T extends string | number>(
     label: string | React.ReactElement,
     value: T,
-    isEnabled: boolean = true,
+    isDisabled: boolean = true,
```

Comment:
I think I prefer your implementation `isDisabled` because html options use the `disabled` flag.  However, we'll need to review any other places this logic may need to be fixed.  Currently the Editor app only shows enabled values in many of the select components.